website/pages/en/index.js

```diff
@@ -14,41 +14,41 @@ const siteConfig = require(process.cwd() + "/siteConfig.js");
 const translate = require("../../server/translate.js").translate;
 
 const pre = "```";
-const typeAnnotationExample= `${pre}hack
-<?hh
-class MyClass {
-  const int MyConst = 0;
-  private string $x = '';
-  public function increment(int $x): int {
-    $y = $x + 1;
-    return $y;
+const asyncExample= `${pre}hack
+  async function helloWorld(): Awaitable<string> {
```

Comment:
We should probably follow Hack naming conventions for top-level functions :)

```suggestion
  async function hello_world(): Awaitable<string> {
```

Reply:
🤭